Core/include/Acts/Detector/LayerStructureBuilder.hpp

```diff
@@ -56,14 +57,6 @@ class LayerStructureBuilder : public IInternalStructureBuilder {
     std::optional<Transform3> transform = std::nullopt;
   };
 
-  /// @brief The surface binning definition
-  struct Binning {
-    /// Define the binning of the surfaces
-    BinningData data;
-    /// An expansion for the filling (in bins)
-    size_t expansion = 0u;
-  };
-
```

Comment:
This is removed and reappears as a proto binning which in term is used now by the LayerStructureBuilder and the GridVolumeBuilder.